### PR TITLE
[PPR-368] CRUD PSP Offers

### PR DESCRIPTION
**Pull request by aomegax**
#### List of Changes
- getOffers
- sendBundleOffer
- removeBundleOffer

- draft of postman collection

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,11 @@
 		<java.version>11</java.version>
 		<cosmos-data-version>3.21.0</cosmos-data-version>
 		<modelmapper-version>2.4.5</modelmapper-version>
+<<<<<<< HEAD
```
Comment (jacopocarlini) [inline pom.xml:20]: fix this

**Review by MarcoAGreco — DISMISSED** [review]
👌

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (aomegax) [reply pom.xml]: done

**Review by jacopocarlini — APPROVED** [review]
great work!

**Pull request merged**